server/game/cards/attachments/00/supportofsaltcliffe.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SupportOfSaltcliffe extends DrawCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: ability.effects.addKeyword('stealth')
+        });
+
+        this.reaction({
+            when: {
+                afterChallenge: ({challenge}) => this.controller === challenge.winner && challenge.isParticipating(this.parent) && challenge.isUnopposed()
```

Comment:
If I recall, @cryogen wanted us to move away from destructuring function parameters (and I agree it's nasty). It only went in initially because we needed to convert 100 usages at once. Best not to introduce more.